frontend/shared/src/server/ServerDocument.tsx

```diff
@@ -0,0 +1,33 @@
+import * as Sentry from '@sentry/browser';
+import Document, {
+  DocumentProps,
+  Head,
+  Html,
+  Main,
+  NextScript,
+} from 'next/document';
+import React from 'react';
+
+process.on('unhandledRejection', (err) => {
+  Sentry.captureException(err);
+});
+
+process.on('uncaughtException', (err) => {
+  Sentry.captureException(err);
+});
+
+class ServerDocument extends Document<DocumentProps> {
+  render(): React.ReactElement {
```

Comment:
Doesn't TS infer the types here?

Reply:
ESLint: Missing return type on function.(@typescript-eslint/explicit-function-return-type)